docs/userguide/transfer_learning.md

```diff
@@ -1,4 +1,51 @@
 # Transfer Learning
 
-This page will soon contain information about transfer learning.
-In the meantime, please see the [examples](../../examples/examples) instead.
\ No newline at end of file
+BayBE offers the possibility to mix data from multiple campaigns in order to accelerate
+optimization. 
+Using data from multiple campaigns is currently supported when using a Gaussian Process
+Surrogate model and is implemented by the [`TaskParameter`](baybe.parameters.categorical.TaskParameter).
+
+```{admonition} Terminology
+:class: note
+The term "Transfer Learning" is used in a lot of different ways.
+Within BayBE, "Transfer Learning" refers to combining data from multiple campaigns.
+Depending on the field, this might also be known as "contextual learning".
+Further note that the act of combining the data of several contexts can be done via
+multiple models, shared architectures, special kernels in a single model, etc.
+We do not necessarily want to limit to any of these methods, even though BayBE currently
+offers only a single one.
+```
+
+## The role of `TaskParameter`
+
+The `TaskParameter` is used to "mark" the context of an individual experiment. The
+set of all possible contexts is provided upon the initialization of a `TaskParameter`
+by providing them as `values`.
+
+```python
+from baybe.parameters import TaskParameter
+
+TaskParameter(name="Month", values=["Nov23", "Dec23", "Jan24"])
+```
+
+If not specified further, a campaign using the `TaskParameter` as specified above
+would now make recommendations for all possible values of the parameter. Using the
+`active_values` argument upon initialization, this behavior can be changed such that
+the `campaign` only makes recommendations for the corresponding values.
+
+```python
+from baybe.parameters import TaskParameter
+
+TaskParameter(name="Month", values=["Nov23", "Dec23", "Jan24"], active_values=["Jan24"])
+```
+
+This can be abstracted easily to other scenarios such as changing substrates (while
+screening same reaction conditions) or formulating mixtures for different cell lines:
+
+~~~python
+TaskParameter(name="Substrate", values=["3,5-dimethylisoxazole", "benzo[d]isoxazole", "5-methylisoxazole"], active_values=["3,5-dimethylisoxazole"])
+TaskParameter(name="Month", values=["Liver cell", "Heart cell", "Hamster brain cell"], active_values=["Liver cell"])
+~~~
+
+A full example demonstrating BayBE's transfer learning capabilities can be found
```

Comment:
where is the picture? imo worth a section

Reply:
Have included a very tiny bit of text here and the two pictures, is this good now?

Reply:
ilike